src/system/RoutesModule/Api/Base/AdminApi.php

```diff
@@ -14,12 +14,12 @@
 
 use ModUtil;
 use SecurityUtil;
-use Zikula_AbstractApi;
+use Zikula\Core\Api\AbstractApi;
```

Comment:
Yes, but the Most generator still uses it. This is why its marked WIP.

Craig Heydenburg <notifications@github.com> schrieb am Sa., 5. Sep. 2015
20:06:

> In src/system/RoutesModule/Api/Base/AdminApi.php
> <https://github.com/zikula/core/pull/2579#discussion_r38813219>:
>
> > @@ -14,12 +14,12 @@
> >
> >  use ModUtil;
> >  use SecurityUtil;
> > -use Zikula_AbstractApi;
> > +use Zikula\Core\Api\AbstractApi;
>
> you just voted to eliminate this class....
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zikula/core/pull/2579/files#r38813219>.
>